MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+* To display the item details into another view.
+* To expand the view like, accordion.
+* To change the underlying bound data.
+* To skip other events like selection events if the Handled property set to `true`.
+
+### ItemDoubleTapped event
+
+The [ItemDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDoubleTapped) event will be triggered whenever double tapping the item. The [ItemDoubleTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html) has the following members providing information for the `ItemDoubleTapped` event:
+
+ * [ItemType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html#Syncfusion_Maui_ListView_ItemDoubleTappedEventArgs_ItemType): It gets the type of double tapped item.
+ * [ItemData](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html#Syncfusion_Maui_ListView_ItemDoubleTappedEventArgs_ItemData): The underlying data associated with the double tapped item as its arguments.
+ * [Position](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html#Syncfusion_Maui_ListView_ItemDoubleTappedEventArgs_Position): Gets the touch position in the double tapped item.
+
+{% tabs %}
+{% highlight c# %}
+
+listView.ItemDoubleTapped += ListView_ItemDoubleTapped;
+
+private void ListView_ItemDoubleTapped(object sender, Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs e)
+{
+    var listViewInboxInfo = new ListViewInboxInfo();
+    listViewInboxInfo.Title = "Bryce Thomas";
+    listViewInboxInfo.Subject = "Congratulations on the move!";
+    viewModel.InboxInfo.Add(listViewInboxInfo);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The [ItemDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDoubleTapped) event is used for the following use cases:
+
+* To show the context menu.
+* To delete the item in the list view at runtime.
+* To change the underlying bound data.
+
+### ItemLongPress event
+
+The [ItemLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemLongPress) event will be triggered whenever long pressing the item. Here, [LongPressCommandParameter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_LongPressCommandParameter) sets the parameter for [SfListView.LongPressCommand](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_LongPressCommand) and its default value is [ItemLongPressEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemLongPressEventArgs.html). `ItemLongPressEventArgs` has the following members which provides the information for `ItemLongPress` event:

```

Comment:
`ItemLongPressEventArgs` has the following members which provide information for `ItemLongPress` event: